### PR TITLE
Implemented 'Favorite Spiders'

### PR DESCRIPTION
**Pull request by pohribeduard**

### CODE REVIEW

File: ScrapyKeeper/app/templates/job_favorites.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            }
+        })
+});
+</script>
+
+{% endblock %}
+{% block script %}
+
+
+{% endblock %}
```
[the remark this answers is not in the document]
Comment (georgetutuianu) [reply ScrapyKeeper/app/templates/job_favorites.html:75]: I'm thinking that we should render only the table with the info when we're doing the ajax request.
To do that, I think that we should make a `POST` request. This way, we can check in python if it's a `GET` request, we can only render the layout and if it's a `GET` request, we can only render the table.

**Comment by georgetutuianu** [comment]
When someone is adding a vehicle spider as favorite, the `seller` for that spider should be added automatically as a favourite also.
Similarly, if some is adding a seller as a favorite, the vehicle spider should also be added automatically.

The same thing should happen when someone is removing a spider.

**Comment by pohribeduard** [comment]
@georgetutuianu done

**Comment by georgetutuianu** [comment]
Great job so far, @pohribeduard !
I have two more improvements that came to me while testing this feature:

1. It would make more sense if `My spiders` would be under the `Spiders` menu. Currently, it is under the `Jobs` menu.
2. I found a way to have some structured data in localstorage. We have to use JSONs (in javascript we'll have objects). 
To store the JSON in localstorage we can use something like `window.localStorage.setItem('fav', JSON.stringify({"list": ["a", "b", "c"]}));`  (`{}` is an object in JS)
To grab the list from the localstorage, we can easily do this: `JSON.parse(window.localStorage.getItem('fav'))`

**Pull request merged**